dashboards-reports/server/routes/lib/createReport.ts

```diff
@@ -48,6 +48,9 @@ export const createReport = async (
     request.query.dateFormat || DATA_REPORT_CONFIG.excelDateFormat;
   // @ts-ignore
   const csvSeparator = request.query.csvSeparator || ',';
+  // @ts-ignore
+  const allowLeadingWildcards = request.query.allowLeadingWildcards;
```

Comment:
give it a default value? in case uiSettings is missing or the service failed this would be an empty string and might cause issues
```suggestion
  const allowLeadingWildcards = !!request.query.allowLeadingWildcards;
```